aries/aries_vcx/src/common/primitives/credential_schema.rs

```diff
@@ -44,7 +44,7 @@ impl Schema {
         submitter_did: &Did,
         name: &str,
         version: &str,
-        data: &Vec<String>,
+        data: Vec<String>,
```

Comment:
nitpick: rename to `attrs` or `attributes`

Reply:
Absolutely